src/cecs277TermProject/AmenitiesDecorator.java

```diff
@@ -0,0 +1,16 @@
+package cecs277TermProject;
+
+public abstract class AmenitiesDecorator  extends Room {
+	public AmenitiesDecorator(Room currentRoom, RoachMotelAmenities Amenity) {

```

Comment:
Create a member var to store `currentRoom` so that it can be used in `toString()` and `getCost()`.

Remove unused `RoachMotelAmenities Amenity` parameter.